server/http/helloapp/embedded_wish.go

```diff
@@ -0,0 +1,197 @@
+package helloapp
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (h *helloapp) handleCreateEmbedded(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, data *api.Call) (int, error) {
+	err := h.asBot(func(mmclient *model.Client4, botUserID string) error {
+		post := &model.Post{
+			Message:   "Debug form",
+			ChannelId: data.Context.ChannelID,
+			UserId:    botUserID,
+		}
+		post.AddProp("appID", AppID)
+		post.AddProp("dialog", h.getDialogSmallSample())
```

Comment:
we should decide on the term and be consistent. Perhaps, `"embedded_form"`? also, `constants.XXX`

Reply:
It is the same as an interactive dialog. So we should unify the two concepts. Here some proposals:
- Interactive Dialog - Embedded Dialog
- Modal Form - Embedded Form

Reply:
Why should we ultimately support the old-style interactive dialogs? I thought it was temporary "stub", to be replaced later?